### DISCUSSION TITLE
Data Management - Recipes - Add Search Filter

### DISCUSSION THREAD

**Discussion by mbriney** [mealie-recipes/mealie#2734] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Data manager makes it easy to batch adjust your recipe data to catgories and tags.  However it's not terribly easy to use if you have a lot of recipes.  Adding a filter search would make it easier to search through the data and make updates.  

### Please provide a concise description of the feature that would resolve your issue.

 In the Data Manager > Recipe Data add a filter search like exists in foods, labels and units. 

### Please consider and list out some caveats or tradeoffs made in your design decision

Should be an easy adjustment given that it's an existing UI in the other categories.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by monksy** [comment]
Would it be reasonable to adapt this PR to include search filters for nutiritional data?

**Comment by bbrodka2** [comment]
Plus 1 on this request
It would be great to have filtering available to edit in mass
Filter for source url field would be great too
Also option to delete in mass recipes based on filtered result
